### ISSUE TITLE
CLI Enhancements

### ISSUE THREAD

**Issue by jmeaster30** [jmeaster30/simpledns#12]
- [ ] Add Record
- [ ] Remove Record
- [ ] List Records (Maybe with filters??)
- [ ] Change Settings
    - [ ] Listening port
    - [ ] Passthrough port
    - [ ] thread count
    - [ ] useudp
    - [ ] usetcp
    - [ ] passthrough dns server IPs
- [ ] Add domain name block lists
- [ ] Remove block list

**Comment by witchofthewires** [comment]
so you do already have the capability to change the listening port, i did this through the config file and verified with netstat/nslookup. you also have the capability to turn udp and tcp on and off, but the server currently only works over udp.

**Comment by jmeaster30** [comment]
Yep! I just want to manage the settings through the cli too. I want a cli like this so I can make a plugin for OPNsense which I use for a firewall in my home network

Yeah tcp technically kinda works but it times out on every request for some reason see #8

**Comment by jmeaster30** [comment]
Passthrough port is irrelevant after #32
